tests/lib/cli-engine.js

```diff
@@ -1854,4 +1854,35 @@ describe("CLIEngine", function() {
         });
 
     });
+
+    describe("when evaluating code with comments to change config when allowInlineConfig is disabled", function() {
+
+        it("should report a violation for disabling rules", function() {
```

Comment:
Can you also add a test for the opposite? Make sure without the option that inline comments still work